### DISCUSSION TITLE
luctusjobranks hud

### DISCUSSION THREAD

**Discussion by Buenoabi** [community/community#59791] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hallo,

Können sie vlt zu dem Jobrank system von Luctus ein Hud dazu machen wo stehe welchen rang mann hat und welcher der höste ist von zahl halt? 
Währe echt cool

**Comment by otter-computer** [comment]
Hi @Buenoabi, thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
